docs/source/Htypes.rst

```diff
@@ -0,0 +1,5 @@
+Htypes
+======
+
+.. currentmodule:: hub.htype
```

Comment:
also perhaps we should mention that link and sequences are meta htypes that can modify other htypes, similar to what we wrote in the paper